src/olympia/api/serializers.py

```diff
@@ -60,7 +60,14 @@ def fake_object(self, data):
     def handle_date(self, value):
         if not value:
             return None
-        return datetime.strptime(value, u'%Y-%m-%dT%H:%M:%S')
+
+        # Don't be picky about microseconds here. But we get them some time
+        # so we have to support them. Maybe there's a nicer way of doing
+        # this? (cgrebs)
```

Comment:
Not really nicer, but https://github.com/mozilla/zamboni/blob/master/mkt/search/serializers.py#L9